spec/three_by_three_ttt_rules_spec.rb

```diff
@@ -75,6 +75,32 @@
     end
   end
   
+  describe "#player_won_game?(board, player)" do 
+    
+    it "returns true when the player has won the game" do 
+      board = ThreeByThreeTTTBoard.new(
+        ["X", " ", " ", 
+         " ", "X", " ", 
+         " ", " ", "X"]
+      )
+      player = double("PlayerX", :marker => "X")
+
+      expect(rules.player_won_game?(board, player)).to eq(true)
+    end
+
+    it "returns false when the player has not won the game" do 
+      board = ThreeByThreeTTTBoard.new(
+        ["X", " ", " ", 
+         " ", "X", " ", 
+         " ", " ", "X"]
+      )
+      player = double("PlayerX", :marker => "O")
```

Comment:
Using a real `Player` or `instance_double` will ensure that you get an error if you decide to rename `:marker`.